File: Java/SheduledTaskJava/src/Task.java
import java.time.LocalDateTime;

class Task {
    private String description;
    private LocalDateTime executionTime;

    public Task(String description, LocalDateTime executionTime) {
        this.description = description;
        this.executionTime = executionTime;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public LocalDateTime getExecutionTime() {
        return executionTime;
    }

    public void setExecutionTime(LocalDateTime executionTime) {
        this.executionTime = executionTime;
    }

    @Override
    public String toString() {
        return description + " - " + executionTime.toString();
    }
}

File: ScheduledTask/Task.cs
namespace ScheduledTask
{
    public class Task
    {
        private string? description;
        private DateTime executionTime;

        public string? Description
        {
            get { return description; }
            set { description = value; }
        }

        public DateTime ExecutionTime
        {
            get { return executionTime; }
            set { executionTime = value; }
        }

        public Task(string description, DateTime executionTime)
        {
            Description = description;
            ExecutionTime = executionTime;
        }

        public override string ToString()
        {
            return $"{Description} - {ExecutionTime:dd.MM.yyyy HH:mm}";
        }
    }
}